garbagecoll/collector.go

```diff
@@ -0,0 +1,87 @@
+package garbagecoll
+
+import (
+	"context"
+	"github.com/antonbaumann/spotify-jukebox/db"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	SessionExpiration = time.Hour * 48
+	CleaningInterval  = time.Hour
+)
+
+// responsible for deleting inactive sessions
+type garbageCollector struct {
```

Comment:
maybe it's better to export this struct - or if you want to abstract it return an interface in the `New()` function